src/wallet/signer.rs

```diff
@@ -114,7 +114,9 @@ use crate::descriptor::XKeyUtils;
 /// multiple of them
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 pub enum SignerId {
+    /// Bitcoin HASH160 (RIPEMD160 after SHA256) hash of an ECDSA public key
     PkHash(hash160::Hash),
+    /// The first 32 bits of the bitcoin HASH160 hash of an ECDSA public key
```

Comment:
The fingerprint is actually the BIP32 key fingerprint, it's not the hash of a key.

I would just document this as `The fingerprint of a BIP32 extended key `

Reply:
Fixed.. I was a little too proud of myself for looking up the full definition. :-)